styles/styles.css

```diff
@@ -816,15 +816,6 @@ a.button.text-link-with-video {
   padding: 5px;
 }
 
-.icon-play-video {
```

Comment:
I'm not sure about removing this css from `style.css`. It can be used by other blocks.
The class is used in the `embed` block (`embed` block use `addPlayIcon` which adds the `.icon-play-vidoe`).

Reply:
<img width="611" alt="obraz" src="https://github.com/Netcentric/vg-volvotrucks-us-rd/assets/125962117/18e0a8ad-1db7-46e6-8b9a-dbfedf541225">
The play button is not displayed properly anymore
Link: https://180-media-with-text--vg-volvotrucks-us-rd--netcentric.hlx.page/trucks/vnr-electric/